### PR TITLE
Continuous (Automated) Delivery/Deployment using Edeliver on Travis-CI

### PR DESCRIPTION
**Pull request by nelsonic**
Adds script/code for automating deployment to Azure from Travis-CI https://github.com/healthlocker/healthlocker/issues/841

@katbow / @RobStallion This is "Part 1" of Automated Deployment.
we need the `/_version` route to be available on https://staging.healthlocker.uk/ 
`before` I can continue on "Part 2" ... as the "Hot Upgrade" deployment relies on a _known_ Git Version
Please review & merge when you can. thanks! 👍

### CODE REVIEW

**Comment by nelsonic** [comment]
Travis-CI cannot find the `id_rsa` (`private` key) because of `\/` on line 531:
https://travis-ci.org/healthlocker/healthlocker/builds/260107780#L531
image

**Comment by nelsonic** [comment]
@katbow / @RobStallion This is "Part 1" of Automated Deployment.
we need the `/_version` route to be available on https://staging.healthlocker.uk/ `before` the "Part 2" will work. Please review & merge when you can. thanks! 👍

**Comment by nelsonic** [comment]
https://travis-ci.org/healthlocker/healthlocker/builds/260175981#L1268
image
Apparently we need to update the permissions on the `.deliver/version.sh` script ...

**Comment by nelsonic** [comment]
While waiting for PR review I'm trying to crack on with "Part 2" of the deployment automation ...
Still getting a `relup` error:
image

**Comment by nelsonic** [comment]
Build _succeeds_ but we get this _lame_ error when deploying:
image

**Comment by nelsonic** [comment]
`vm.args: No such file or directory` ...
image

Looking into:
http://sgeos.github.io/phoenix/elixir/erlang/ecto/distillery/postgresql/mysql/2016/09/18/storing-elixir-release-configuration-in-environment-variables-with-distillery.html

File: web/controllers/github_version_controller.ex [hunk 1]
```diff
@@ -0,0 +1,21 @@
+defmodule Healthlocker.GithubVersionController do
+  use Healthlocker.Web, :controller
+
+  def index(conn, _params) do
+    # leaving IO.inspect / puts in for debugging till further notice
+    IO.inspect(System.cmd("pwd", []))
```
Comment (katbow) [inline web/controllers/github_version_controller.ex]: do we want to leave the `IO.inspects` in this file? 🤔

**Comment by katbow** [comment]
@nelsonic looks good.

Just to clarify for myself, will PRs merged to HL after this is merged in auto-deploy to azure? I didn't think so since this is 'Part 1'. I wanted to confirm since we don't yet want to deploy until sign off from MW that testing of the app is done.

File: web/controllers/github_version_controller.ex (hunk 1 above)
Comment (nelsonic) [inline web/controllers/github_version_controller.ex]: @katbow yeah, leaving the `IO.inspect` are in for now. will remove when deployment is working _seamlessly_. 👍

**Comment by nelsonic** [comment]
@katbow no, automatic continuous deployment of HL is not yet enabled and will not be until approved.

**Pull request merged**